schema/spacewalk/common/tables/rhnPackageChangeLogData.sql

```diff
@@ -13,6 +13,7 @@
 -- in this software or its documentation.
 --
 
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
```

Comment:
I feel this should go in `schema/spacewalk/postgres/start.sql`. I see we have there already something for dblink there but I'm not sure to understand how that works. Maybe @mcalmer has more clues.